frontend/packages/console-dynamic-plugin-sdk/src/app/core/reducers/core.ts

```diff
@@ -0,0 +1,65 @@
+import { CoreState } from '../../redux-types';
+import { ActionType, CoreAction } from '../actions/core';
+
+/**
+ * Reducer function for the core
+ * @param state the reducer state
+ * @param action provided associated action type alongwith payload
+ * @param action.type type of the action
+ * @param action.payload associated payload for the action
+ * @see CoreAction
+ * @return The the updated state.
+ * * */
+const coreReducers = (
```

Comment:
I'm a little weary about the name. `core` SDK perhaps, `core` to the store or to the SDK as a whole... is it? I do not have a better name. Although it shouldn't be plural.

```suggestion
const coreReducer = (
```

Reply:
yeah, it depends on how granular we would like to go with reducers on a high level we can have two
- core (anything specific to host app)
- k8s 
- something else depending on the need

Reply:
So the app will have it's own reducer (maybe?) and we'll have some core functionality we want to support from the SDK. Likely functionality to help the app do some common stuff. Perhaps `core` is okay here... it just feels weird from a naming point of view. 

I guess if we never really expose this name externally, the naming is fine. If we collect some our reducers in an array and export this "array of opaque reducers" for the hosting app to consume in their Store (if they have one) but we also give them the store via an export (likely via a provider -- not sure we want to expose a store directly) if they don't have redux.

Reply:
Looks like we don't actually expose it, it gets consumed in `baseReducers` and that is then consumed in a local store. Since the `baseReducers` is not exported outside the SDK, I think my above comment is more-or-less moot. We may want to expose this at some point in the future so the host app can install the reducers into their host app's redux store -- but we can put that on the back burner for now.

Reply:
Based on req what we have i.e console/hac, they both will have redux context in host app only so we just need to expose the common reducer and so there are two things 
- Do we need to export the reducer explicitly via SDK, with render flow it'll be accessible with the current change itself.
- If we want to expose it or for that matter, we definitely want to expose `AppInitSDK` then this should not be exposed as core-API's as these are for plugin developers, so what is the right way to expose these for app developers.

Reply:
@invincibleJai that's really the big question... of which we do not need to decide for your PR. What you have done via the `AppInitSDK` is good enough for now. I wonder if we don't want to integrate more cleanly with a host app though, but that discussion isn't needed at this moment.